src/compositions/UserPopup/UserPopup.tsx

```diff
@@ -0,0 +1,34 @@
+import { FC } from 'react';
+
+import { AppRoutes } from '../../routes';
+import IconButton from '../IconButton/IconButton';
+
+import './UserPopup.scss';
+
+interface UserPopupProps {
+  address: string;
+  ENS?: string;
+  onLogoutClick: () => void;
+}
+
+const UserPopup: FC<UserPopupProps> = (
+  {
+    address, ENS, onLogoutClick,
+  },
+) => (
+  <div className="user-popup">
+    <div className="user-popup__address-bar">
+      <div>
```

Comment:
You have a WalletInfo component where you can reuse to display the address and the icon. It would be great to reuse the existing component instead of re create a similiare one. 
You can pass in paramater some option to hide the address image in this case.

Reply:
Right, ill refactor to move these two together